### DISCUSSION TITLE
HM-IP HmIP-FBL

### DISCUSSION THREAD

**Discussion by antoni105** [thkl/hap-homematic#368] [Q&A]
Hallo zusammen!

Habt Ihr Erfahrungen, wie man in Homekit bzw. HAP einen Raffstoren Aktor einbinden kann, sodass die Lamellenposition eingestellt werden kann? Aktuell kann ich die Raffstoren nur hoch und runter fahren.

Vielen Dank!

Gruß
antoni105

**Comment by gerd2005** [comment]
Hallo,

Erfahrungen habe ich nicht, hätte aber vermutet, dass das über die „Lamellenfunktion“ funktioniert.

![6B55F899-65B8-4261-93C7-CAF2EBC376BD](https://user-images.githubusercontent.com/72799068/108357479-c61e6280-71ed-11eb-9eca-857cac4a6e96.png)

**Comment by antoni105** [comment]
Hallo Gerd,

das hätte ich auch vermutet, im HomeKit wird mir allerdings nichts angezeigt.


Gruß 
Antoni105
